code/style.css

```diff
@@ -1,7 +1,551 @@
+  font-family: 'Playfair Display';
+  font-size: 3vw;
+  font-style: normal;
+  font-weight: normal;
+  text-align: center;
+  margin: 4rem 2rem;
+  border-top: 0.5px solid black;
+  border-bottom: 0.5px solid black;
+  padding: 20px 0;
+}
+
+h3 {
+  font-family: 'Poppins';
+  text-transform: uppercase;
+}
+
+h4 {
+  text-align: center;
+  font-size: 15px;
+  font-family: 'poppins';
+  font-weight: normal;
+  padding-bottom: 2rem;
+}
+
+h5 {
+  text-align: center;
+  font-size: 15px;
+  font-family: 'poppins';
+  font-weight: normal;
+  font-size: 20px;
+}
+
+/* Styling of header starts here */
+
+.video-bg {
+  position:absolute;
+  top: 0;
+  left: 0;
+  min-width: 100%;
+  min-height: 100%;
+  z-index: -1;
+}
+
+.header {
+  height: 75vh;
+  color: #f5f3f2;
+  align-items: center;
+  position: relative;
+  overflow: hidden;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+    }
+  
+  nav a:hover {
+    color: black;
+    text-decoration: underline white;
+    transition: all 1s ease;
+  }
+  
+  .logo {
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    background-color:white;
+  }
+
+  /* Mobile nav */
+
+.hamburger-menu {
+  display: flex;
+  flex-direction: column;
+}
+
+/* Hamburger menu */
+
+.menu-line {
+  width: 1.5rem;
+  height: 1px;
+  margin-bottom: 0.5rem;
+  fill: white;
+  background-color: white;
+}
+
+/* Desktop nav */
+
+.desktop-menu {
+  display: none;
+  text-transform: uppercase;
+}
+
+.desktop-menu a {
+  margin-left: 1.5rem;
+}
+
+.hero-title-wrapper {
+  color: black;
+  margin-top: 150px;
+  position: relative;
+}
+
+.hero-title {
+  font-size: 5vw;
+  font-family: 'Playfair Display';
+  font-style: normal;
+  font-weight: normal;
+  text-align: center;
+}
+
+.hero-title-wrapper p {
+  text-align: center;
+}
+
+p {
+  font-family: 'poppins';
+  font-size: 1rem;
+  font-weight: normal;
+}
+
+/* Animated text */
+
+.rolling-text-header {
+  font-family: 'Playfair Display';
+  font-size: 4vw;
+  font-style: normal;
+  font-weight: normal;
+  margin-top: 1rem;
+  animation-name: scroll;
+  animation-duration: 10s;
+  position: relative;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+
+@keyframes scroll {
+  from { left: 100%; }
+  to { left: -80%; }
+}
+
+/* Styling of accordion starts here */
+
+.project-header {
+  border-bottom: 3px solid black;
+}
+
+.active + .project-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.project-image {
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.project-title {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-family: 'poppins';
+  font-size: 10px;
+  padding:10px;
+  height: 70px;
+}
+
+.project-title p {
+  font-size: 10px;
+}
+
+.project-title:hover {
+  background-color: black;
+  color: #EED2D5;
+  transition: all 1.5s ease;
+}
+
+.project-title::before {
```

Comment:
Nice use of pseudo elements! I did not know that we could add "+ " and "-" in the content! Very smart!  Instead, I created 2 divs to make a cross and a minus sign haha! I will definitely apply this technique next time.